include/baresip.h

```diff
@@ -294,6 +294,12 @@ int custom_hdrs_add(struct list *hdrs, const char *name, const char *fmt, ...);
 int custom_hdrs_apply(const struct list *hdrs, custom_hdrs_h *h, void *arg);
 
 
+void debug_list_hdrs( struct list *custom_headers );
```

Comment:
please add a prefix to the function name ...

if the parameter `custom_headers` is not modified, it can be `const`